src/components/Nav/Menu/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { AnimatePresence, motion } from "framer-motion"
+import { Box, type BoxProps, Flex, Text } from "@chakra-ui/react"
+import * as NavigationMenu from "@radix-ui/react-navigation-menu"
+
+import { Button } from "@/components/Buttons"
+
+import { SECTION_LABELS } from "@/lib/constants"
+
+import type { NavSections } from "../types"
+
+import SubMenu from "./SubMenu"
+import { useNavMenu } from "./useNavMenu"
+
+type NavMenuProps = BoxProps & {
+  sections: NavSections
+}
+
+const Menu = ({ sections, ...props }: NavMenuProps) => {
+  const {
+    activeSection,
+    containerVariants,
+    direction,
+    handleSectionChange,
+    isOpen,
+    menuColors,
+    onClose,
+  } = useNavMenu(sections)
+
+  return (
+    <Box {...props}>
+      <NavigationMenu.Root
+        dir={direction}
+        orientation="horizontal"
+        delayDuration={500}
+        onValueChange={handleSectionChange}
+      >
+        <NavigationMenu.List asChild>
+          <Flex as="ul" listStyleType="none">
```

Comment:
Nice to have: use `UnorderedList` component from chakra. Similar as in the SubMenu